index.bs

```diff
@@ -4092,23 +4082,15 @@ Revising a Recommendation: Substantive Changes</h5>
 <h5 id="revised-rec-features">
 Revising a Recommendation: New Features</h5>
 
-	Tentative new features (see <a href="#correction-classes">class 4 changes</a>)
-	<em class=rfc2119>may</em> be annotated into a [=Recommendation=]
-	explicitly identified as [=allow new features|allowing new features=]
-	using [=candidate additions=].
-
-	Note: [=Candidate additions=] do not normatively modify the document;
-	they editorially indicate how one might do so.
-	They are therefore published following the provisions of [[#revised-rec-editorial]].
-
-	A [=candidate addition=] can be made normative
-	and be folded into the main text of the [=Recommendation=]
-	using the same process as for [=candidate corrections=],
-	as detailed in [[#revised-rec-substantive]].
+	For [=Recommendations=] explicitly identified as  [=allow new features|allowing new features=],
+	tentative new features (see <a href="#correction-classes">class 4 changes</a>)
+	<em class=rfc2119>may</em> be annotated in as [=candidate additions=],
+	and <a href="#correction-classes">class 4 changes</a> may be normatively incorporated
+	as for <a href="#correction-classes">class 3 changes</a> in [[#revised-rec-substantive]].
```

Comment:
```suggestion
	For [=Recommendations=] explicitly identified as [=allow new features|allowing new features=],
	tentative new features (see <a href="#correction-classes">class 4 changes</a>)
	<em class=rfc2119>may</em> be annotated as [=candidate additions=],
	and <a href="#correction-classes">class 4 changes</a> may be normatively incorporated
	in the same fashion as <a href="#correction-classes">class 3 changes</a> in [[#revised-rec-substantive]].
```

Reply:
2 of the 3 suggested tweaks have been applied by https://github.com/w3c/process/commit/5a60ea5afefb2f642a715925feab93b655b3cfa3, and I've opened a new issue to discuss the third in https://github.com/w3c/process/issues/899